__tests__/app.test.js

```diff
@@ -42,23 +42,24 @@ describe("/api", () => {
 
 describe("api/articles/:article_id", ()=>{
   test("200: returns the article with the associated article_id", ()=>{
-    //const articleId = 1
       return request(app)
-      .get("/api/articles/1")
+      .get("/api/articles/2")
       .expect(200)
       .then(({ body })=>{
           const {article} = body
-          expect(article).toEqual([{
-            article_id: 1,
-            title: "Living in the shadow of a great man",
-            topic: "mitch",
-            author: "butter_bridge",
-            body: "I find this existence challenging",
-            created_at: "2020-07-09T20:11:00.000Z",
-            votes: 100,
-            article_img_url:
-              "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
-          }])
+          expect(article).toEqual([
+          {
+            article_id: 2,
+            title: 'Sony Vaio; or, The Laptop',
+            topic: 'mitch',
+            author: 'icellusedkars',
+            body: 'Call me Mitchell. Some years ago—never mind how long precisely—having little or no money in my purse, and nothing particular to interest me on shore, I thought I would buy a laptop about a little and see the codey part of the world. It is a way I have of driving off the spleen and regulating the circulation. Whenever I find myself growing grim about the mouth; whenever it is a damp, drizzly November in my soul; whenever I find myself involuntarily pausing before coffin warehouses, and bringing up the rear of every funeral I meet; and especially whenever my hypos get such an upper hand of me, that it requires a strong moral principle to prevent me from deliberately stepping into the street, and methodically knocking people’s hats off—then, I account it high time to get to coding as soon as I can. This is my substitute for pistol and ball. With a philosophical flourish Cato throws himself upon his sword; I quietly take to the laptop. There is nothing surprising in this. If they but knew it, almost all men in their degree, some time or other, cherish very nearly the same feelings towards the the Vaio with me.',
+            created_at: '2020-10-16T05:03:00.000Z',
+            votes: 0,
+            article_img_url: 'https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700',
+            comment_count: '0'
+          }
+          ])
```

Comment:
If following TDD practices, when adding a new feature to your software you should be adding _new tests_ ✅ and not _changing old ones_ 🚫 
This test would have needed to be changed anyway, as it was using `toEqual` so it would have failed as you added in a new property. This is where a matcher like `toMatchObject` is better because it allows for extra properties on the object, alternatively it could have tested each property one at a time.
Nevertheless, your test works, but it might be better to pick an article has does have comments ( and maybe with a smaller `body` ) for the request 🤔